### DISCUSSION TITLE
What are the default timeouts?  Are they configurable?

### DISCUSSION THREAD

**Discussion by NathanielWroblewski** [oakserver/oak#498] [Q&A]
It seems Deno's http server [doesn't yet surface timeouts](https://github.com/denoland/deno/issues/3515), but this is an important feature for server scalability and performance.  How are timeouts canonically implemented in oak?  Do you have an example of this?  Relatedly, what are the default values, and are they configurable?

**Comment by kitsonk** [comment]
oak is a middleware framework that overlays an HTTP Server.  It does not introduce anything over and above for connection timeouts than what is provided by Deno.
